examples/src/main/java/kafka/examples/Consumer.java

```diff
@@ -16,52 +16,40 @@
  */
 package kafka.examples;
 
-
-import java.nio.ByteBuffer;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.Collections;
 import java.util.Properties;
-import kafka.consumer.ConsumerConfig;
-import kafka.consumer.KafkaStream;
-import kafka.javaapi.consumer.ConsumerConnector;
-import kafka.message.MessageAndMetadata;
 
+import org.apache.kafka.clients.consumer.ConsumerRecord;
+import org.apache.kafka.clients.consumer.ConsumerRecords;
+import org.apache.kafka.clients.consumer.KafkaConsumer;
 
 public class Consumer extends Thread
```

Comment:
Could we extends from kafka.utils.ShutdownableThread instead to allow it being able to gracefully interrupted and closed?